src/Pagination/components/PaginationContainer.ts

```diff
@@ -39,7 +39,7 @@ interface PaginationPageState {
 }
 
 class PaginationContainer extends Component<ModelerProps, PaginationContainerState> {
-    private widgetDom: HTMLElement | null = null;
+    private widgetDom: Element | null = null;
```

Comment:
Maybe the same change is necessary in "connectToListView" function.

Reply:
Does not depend on context, validation is based on props not on state